sentry/src/main/java/io/sentry/SentryOptions.java

```diff
@@ -1118,6 +1125,34 @@ public void setEnableExternalConfiguration(boolean enableExternalConfiguration)
     this.enableExternalConfiguration = enableExternalConfiguration;
   }
 
+  /**
+   * Returns tags applied to all events and transactions.
+   *
+   * @return the tags map
+   */
+  public @NotNull Map<String, String> getTags() {
+    return tags;
+  }
+
+  /**
+   * Sets tags applied to all events and transactions.
+   *
+   * @param tags the tags map
+   */
+  public void setTags(final @NotNull Map<String, String> tags) {
+    this.tags = tags;
```

Comment:
one could set `null` or a non-thread-safe list, what's about only exposing the `setTag(key, value)`?